workflow-templates/hybrid-ci.yml

```diff
@@ -52,31 +52,65 @@ on:
 
       auto-promote:
         description: Whether to auto promote the package
-        value: "false"
+        value: 'false'
 
 permissions:
   id-token: write # This is required for requesting the ID token for AWS actions
   contents: read # This is required for actions/checkout
 
 concurrency:
-  group: "${{ github.workflow }} @ ${{ github.event.pull_request.head.label || github.head_ref || github.ref }}"
+  group: '${{ github.workflow }} @ ${{ github.event.pull_request.head.label || github.head_ref || github.ref }}'
   cancel-in-progress: ${{ github.ref != 'refs/heads/release' }}
 
 env:
   COMMIT_SHA1: ${{ github.sha }}
-  DD_AGENT_MAJOR_VERSION: "7"
-  DD_API_KEY: "${{ secrets.DD_API_KEY }}"
-  DD_APP_KEY: "${{ secrets.DD_APP_KEY }}"
+  DD_AGENT_MAJOR_VERSION: '7'
+  DD_API_KEY: '${{ secrets.DD_API_KEY }}'
+  DD_APP_KEY: '${{ secrets.DD_APP_KEY }}'
   DD_SITE: datadoghq.com
-  DEVOPS_SCRIPTS_PATH: "./devops-utility-belt/scripts"
+  DEVOPS_SCRIPTS_PATH: './devops-utility-belt/scripts'
   GIT_BASE_BRANCH: ${{ github.base_ref }}
   GIT_BRANCH: ${{ github.ref_name }}
-  GITHUB_TOKEN: "${{ secrets.CI_USER_REPO_ACCESS_TOKEN }}"
+  GITHUB_TOKEN: '${{ secrets.CI_USER_REPO_ACCESS_TOKEN }}'
   RELEASE_FULLNAME: $PACKAGE_NAME
   REPO_NAME: ${{ github.event.repository.name }}
+  NAMESPACE: << REPLACE WITH PROJECT NAMESPACE >>
```

Comment:
Can we pull the namespace out of the sfdx-project.json instead?

Reply:
Ah, so it wouldn't have access to the sfdx-project.json. Is that right?

Reply:
https://github.com/ncino/.github/pull/24/files#diff-91bc36802f73c95624da2b6d3c2f86e2bd67f6008a91c7dd121f778da5ad4fbbR89-R96

These lines have it pull down the repo with a sparse checkout on the `sfdx-project.json` for efficiency's sake and then run a jq query to grab it from there

Reply:
i thought i removed it